packages/wonder-blocks-dropdown/src/components/select-opener.tsx

```diff
@@ -183,7 +190,7 @@ export default class SelectOpener extends React.Component<
                 <LabelMedium style={styles.text}>
                     {/* Note(tamarab): Prevents unwanted vertical
                                 shift for empty selection */}
-                    {children || "\u00A0"}
+                    {children || <span aria-hidden="true">&nbsp;</span>}
```

Comment:
I'm curious about this update! Was it causing issues?

Reply:
Yes -- the word "space" was being announced in VoiceOver!

Reply:
Ah good catch! Thank you!